src/content/docs/tutorial-service-level-mgmt/establish-output-sli.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Establish an output performance SLI"
+metaDescription: "New Relic observability maturity series: our service level management guide shows you how to easily measure and communicate the overall health, performance, and quality of your digital products and services to all stakeholders."
+redirects:
+freshnessValidatedDate: never
+---
+
+import apmOverviewScreen from 'images/apm_screenshot-full_overview-screen.webp'
+
+import apmServiceBoundary from 'images/apm_screenshot-crop_service-boundary.webp'
+
+import apmLatencyBaseline from 'images/apm_screenshot-full_latency-baseline.webp'
+
+import apmSLStartFromApm from 'images/apm_screenshot-full_sl-start-from-apm.webp'
+
+The ability to manage and query vast amounts of data efficiently is one of the core capabilities of New Relic. We measure query output health the same as you might measure your client-facing capabilities: by how fast and error-free HTTP query requests receive responses. We refer to this as output performance, and you can measure use SLIs to track this. Once you set up an output performance SLI using the procedure below, you can track both response times and the error rate of your applications.
+
+<Steps>
+    <Step>
+## Identify your service
+
+Make sure that you've done the following:
+* Instrumented your primary applications with [New Relic APM agents](/docs/apm/new-relic-apm/getting-started/introduction-apm).
+* Named your applications with a familiar naming convention, as outlined in our [Service characterization guide](/docs/new-relic-solutions/observability-maturity/operational-efficiency/sc-implementation-guide/#improvement-process).
+* Located your application in our [entity explorer](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts#find).
+
+In the entity explorer, find your application (entity type of `Services - APM`) and select it. You should see the overview screen below. **Don't click `Service levels` yet.**
+
+<img
+  alt="APM Overview screenshot"
+  src={apmOverviewScreen}
+/>        
+    </Step>
+    <Step>
+## Identify your service boundary [#identifying-service-boundary]
+
+Next, you'll want to ensure you're measuring the output of your service. While the dependencies of an application each play a part in response times and success rates, you can measure the final and total response times and success at the point where the request is received and responded to.
```

Comment:
```suggestion
Next, you'll want to ensure you're measuring the output of your service. While the dependencies of an application each play a part in response times and success rates, you can measure the final and total response times and success  rates at the point where the request is received and responded to.
```

Reply:
Hmm, i'm not sure how to find the service map based on these instructions. I would explicitly guide users to the service maps, then describe that the node on the far right (?) will typically be your endpoint. (Idk if this is true, but something that generally describes how to find the service map and what to do once you get there)

Reply:
oh wait i read the second paragraph. i actually have no idea how to interpret the service map part of this, so i got kind of lost. I might chat with a second tech writer and see if this is just a me thing or if there's something we can add to describe "how you can tell"